diff --color --unified --recursive --text package.orig/ubuntu-wallpapers-22.04.4/setup.py package.new/ubuntu-wallpapers-22.04.4/setup.py
--- package.orig/ubuntu-wallpapers-22.04.4/setup.py	2022-06-28 16:59:20.711660124 +0200
+++ package.new/ubuntu-wallpapers-22.04.4/setup.py	2022-06-28 17:02:04.586632861 +0200
@@ -15,6 +15,7 @@
 setup(
     name = 'ubuntu-wallpapers',
     version = version,
+    packages=[],
     data_files=[('share/backgrounds', glob.glob('*.png')+glob.glob('*.jpg')),
 		('share/backgrounds/contest', glob.glob('contest/*.xml')),
                ],
